frontend/views/GroupDashboard.vue

```diff
@@ -8,9 +8,6 @@
       <h1 class="title is-1 is-marginless" data-test="groupName">
         {{ currentGroupState.groupName }}
       </h1>
-      <p data-test="sharedValues">
-        {{ currentGroupState.sharedValues }}
-      </p>
```

Comment:
Looks like removing this breaks the tests.

Here are my thoughts in general on this PR, and I could also use some help from @pieer and @mmbotelho on this: should this issue maybe be incorporated as part of a larger PR to implement the radically changed UI that @mmbotelho designed on Figma? Or should it be merged as is with fixed tests? What do you guys think?

Reply:
I would say that merging is better than waiting for a massive PR to be done. I created these issues with that in mind - doing small pieces of code and merging them as fast as possible. @pieer what do you think?

Reply:
@taoeffect Yes fix the test and merge, I will update the full page soon anyway
@mmbotelho Breaking issues into small tasks is better indeed